MM/6.1.0/config/media_manager_5/labels/Cache Manager.dcl

```diff
@@ -0,0 +1,640 @@
+    {
+      default_translation = 'Refresh'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Opdater siden'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label embedded_cache_manager_results_per_page {
+  key = 'EMBEDDED_CACHE_MANAGER_RESULTS_PER_PAGE'
+  group = 'Cache Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Results per page'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Resultater per side'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label embedded_cache_manager_loading_text {
+  key = 'EMBEDDED_CACHE_MANAGER_LOADING_TEXT'
+  group = 'Cache Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Loading...'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Indlæser...'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label embedded_cache_manager_custom_format {
+  key = 'EMBEDDED_CACHE_MANAGER_CUSTOM_FORMAT'
+  group = 'Cache Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Custom'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Brugerdefineret'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label embedded_cache_manager_manageable_actions_bar_delete_button {
+  key = 'EMBEDDED_CACHE_MANAGER_MANAGEABLE_ACTIONS_BAR_DELETE_BUTTON'
+  group = 'Cache Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Delete'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Slet'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label embedded_cache_manager_remove_cached_files_title {
+  key = 'EMBEDDED_CACHE_MANAGER_REMOVE_CACHED_FILES_TITLE'
+  group = 'Cache Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Delete'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Slet'
+      language_id = data.language.danish.id
+    }
+  ]
+}
+
+resource configservice_label embedded_cache_manager_remove_cached_files_content {
+  key = 'EMBEDDED_CACHE_MANAGER_REMOVE_CACHED_FILES_CONTENT'
+  group = 'Cache Manager'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Are your sure you want to delete {multiple, select, true {these cached assets} false {this cached asset}}? This will delete all associated renditions'
```

Comment:
```suggestion
      default_translation = 'Are your sure you want to delete {multiple, select, true {these cached assets} false {this cached asset}}? This will delete all associated renditions from your local storage'
```